manifests/resource/mailhost.pp

```diff
@@ -77,13 +75,13 @@
 #  }
 #
 define nginx::resource::mailhost (
-  Integer $listen_port,
+  Stdlib::Port $listen_port,
```

Comment:
Since these are ports, you can drop the `+ 0` later on in the code. That used to be needed when it could have been a String.

Reply:
Done!